format/date.js

```diff
@@ -4,16 +4,18 @@ import {luxonIANAZonesByProfile as timezones} from '../lib/luxon-timezones.js'
 // todo: change to `(profile) => (when) => {}`
 const formatDate = (profile, when) => {
 	let timezone
-	if (timezones.has(profile)) timezone = timezones.get(profile)
-	else {
+	if (timezones.has(profile)) {
+		timezone = timezones.get(profile)
+	} else {
 		timezone = new IANAZone(profile.timezone)
 		timezones.set(profile, timezone)
 	}
 
-	return DateTime.fromMillis(+when, {
+	return DateTime.fromMillis(Number(when), {
 		locale: profile.locale,
-		zone: timezone
-	}).toFormat('yyyyMMdd')
+		zone: timezone,
+	})
+		.toFormat('yyyyMMdd')
```

Comment:
Is reasonably easy to either a) indent *all* chained function calls or b) *none*?

Reply:
That rule caused that rule: <https://eslint.style/rules/js/newline-per-chained-call>

a) As I understand it, it is currently set as this. The new line is created for all chains. Even if the chain only has one element. The default is from 2 elements.

b) We could switch off the rule, so that all variants would be possible, but this could make the code inconsistent.
Setting the rule so that all chains, no matter how long they are, are in one line (what I understand by b), I don't think works. And I know no other suitable rule for this.

Reply:
> a) As I understand it, it is currently set as this. The new line is created for all chains. Even if the chain only has one element. The default is from 2 elements.

Let's try enabling it for >=1 elements then.

Reply:
Okay, the rule is set now this way:

`"@stylistic/newline-per-chained-call": ["error", { "ignoreChainWithDepth": 1 }],`

Reply:
The `.fromMillis()` in `DateTime.fromMillis().toFormat()` is not indented. My idea was to put *every* call in a chain on its own line as soon as there are >1 calls in the chain.
I don't mind wether it's indented or not.

Reply:
Okay, I didn't expect that. But I took another look at the description of the rule. The following examples are listed as OK:

```js
// Or
_
  .chain({})
  .map(foo)
  .filter(bar);

// Or
_.chain({})
  .map(foo)
  .filter(bar);
```

I have not yet found a suitable setting or rule for automatically forcing the first linebreak.

I have now adjusted it manually, but that doesn't quite fit in with what we want to achieve with the use of a linter.

Reply:
Still, it's an improvement over the un-linted status quo!